packages/driver/src/cypress/error_messages.coffee

```diff
@@ -1074,20 +1071,20 @@ module.exports = {
       URL from the `options` object: {{optionsUrl}}
       URL from the `url` parameter: {{url}}
     """
-    cannot_visit_2nd_domain: """
-      #{cmd('visit')} failed because you are attempting to visit a second unique domain.
+    cannot_visit_different_origin: """
+      #{cmd('visit')} failed because you are attempting to visit a URL that is of a different origin.
 
-      You may only visit a single unique domain per test.
+      The new URL is considered a different origin because the following parts of the URL are different: {{differences}}
```

Comment:
maybe this could be improved to be a little more natural-sounding with 1 item, but eh, this is probably clear enough

Reply:
The error messages aren't great to work with as are and need the error improvements PR to make this a function which could actually be manipulated easier to write this more natural. So, this is good as is.